### ISSUE TITLE
Kettering: webform export is blank

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1561]
From Zendesk: Exporting of form data: Entries display in form entries database, but when attempt made to export excel file contains no data. Specific example form: Neighborhood Pride Nominations

I was able to replicate with the form mentioned above: https://www.ketteringoh.org/wp-admin/admin.php?page=gf_entries&id=25

There are entries but the export yields a blank csv file:
https://docs.google.com/spreadsheets/d/1irdjHgTPii3a7RHB53lWVVblMq_RAAl2PRCgIvduIXk/edit#gid=399858011

@jlyon @aschmoe

**Comment by jlyon** [comment]
@kevindherman I was able to replicate this earlier, but I just tried downloading the export and was able to make it work.  I'll email you the file.

**Comment by jlyon** [comment]
Able to replicate on Kettering, but not Beta, SR, or my local.  Needs more digging, @kevindherman

**Comment by kevindherman** [comment]
@jlyon thanks for the update, I'll relay the message.

**Comment by jlyon** [comment]
@kevindherman I was able to import Kettering locally and export the Neighborhood Pride Form. I just emailed it to you.

The response we're getting from the export seems pretty weird. I'm going to look into this again after we push the release tomorrow with hopes that that will fix it.

**Comment by kevindherman** [comment]
@jlyon I just got a zendesk ticket from Rebecca at SR of a similar experience. 

"I just tried to export the entries for the "Technology Vendor Registration" (select form, select all fields, click the export button) on the City of San Rafael main website, but the csv file that was created is blank."

I was able to replicate this as well.

**Comment by jlyon** [comment]
https://legacy.forums.gravityhelp.com/topic/export-returns-empty-cdv-file

**Comment by jlyon** [comment]
@kevindherman I tried digging in, but couldn't figure out what the issue could be. I created a support ticket w Gravityforms.

**Comment by kevindherman** [comment]
@jlyon thanks, I'll let RW know. Do you think we'll eventually be able to get that data or is it potentially lost?

**Comment by jlyon** [comment]
I think we should be able to get it to work.

**Comment by jlyon** [comment]
@kevindherman I was able to get to the bottom of it.  In order to improve performance, we had set a couple of sites (Kettering, SR) up on multiple servers.  These exports save a file on a server, but then the request to download the file was made on another server and was coming up empty. 

To fix this, I set the number of replicas on SR to 1 and was able to download the export from https://www.cityofsanrafael.org/wp-admin/admin.php?page=gf_export.  I'll keep an eye on the performance of SR, and may need to re-increase the number of replicas in the future.
